docs/setup.md

```diff
@@ -317,9 +317,10 @@ to your real kubelet instance's configuration:
 
 ## Logging and Debugging
 
-You can control logging and debugging with the `--logger-*` command line options. By
-default debug logs are globally disabled. You can turn on full debug logs with the
-`--logger-debug '*'` command line option.
+You can control logging with the klog command line options or by setting the
+corresponding `LOGGER_*` environment variables. Additionally, the `LOGGER_DEBUG`
+environment variable controls debug logs. These are globally disabled by default.
+You can turn on full debugging by setting `LOGGER_DEBUG='*'`.
```

Comment:
A bit out-of-scope of this PR, but, should we mention the dynamic configuration here. I.e. `logger:` options can be changed dynamically at runtime

Reply:
Yes, definitely. I'll update the docs.

Reply:
Updated the documentation to describe the precedence of the environment vs. configuration files and ConfigMaps.